addons/crm/views/crm_team_views.xml

```diff
@@ -45,18 +45,6 @@
                     'default_user_id': uid,
                 }
             </field>
-            <field name="help" type="html">
```

Comment:
Set it void to ease migration / update (``<field name="help" type="html"></field>``)

Reply:
it raises view validation error.

Reply:
Maybe don't use "type" and set to void ? If type is html maybe he tries to validate a "not html" content.